[2023-09-17T14:05:31.826+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.dbtRun manual__2023-09-17T14:00:23.768488+00:00 [queued]>
[2023-09-17T14:05:31.834+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.dbtRun manual__2023-09-17T14:00:23.768488+00:00 [queued]>
[2023-09-17T14:05:31.835+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-09-17T14:05:31.852+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): dbtRun> on 2023-09-17 14:00:23.768488+00:00
[2023-09-17T14:05:31.862+0000] {standard_task_runner.py:57} INFO - Started process 479 to run task
[2023-09-17T14:05:31.865+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'etl', 'dbtRun', 'manual__2023-09-17T14:00:23.768488+00:00', '--job-id', '142', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpulanm38k']
[2023-09-17T14:05:31.867+0000] {standard_task_runner.py:85} INFO - Job 142: Subtask dbtRun
[2023-09-17T14:05:31.919+0000] {task_command.py:415} INFO - Running <TaskInstance: etl.dbtRun manual__2023-09-17T14:00:23.768488+00:00 [running]> on host 930ac20b04d8
[2023-09-17T14:05:32.275+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='dbtRun' AIRFLOW_CTX_EXECUTION_DATE='2023-09-17T14:00:23.768488+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-17T14:00:23.768488+00:00'
[2023-09-17T14:05:32.277+0000] {logging_mixin.py:151} INFO - Executando o comando <cd /opt/***/dbt/etl && dbt run -t prod>
[2023-09-17T14:05:39.108+0000] {logging_mixin.py:151} INFO - Resultado: 14:05:35  Running with dbt=1.6.2
14:05:35  Registered adapter: postgres=1.6.2
14:05:35  Unable to do partial parsing because config vars, config profile, or config target have changed
14:05:35  Unable to do partial parsing because profile has changed
14:05:37  [WARNING]: Did not find matching node for patch with name 'str_datas' in the 'models' section of file 'models/staging/despesas/stg_despesas.yml'
14:05:37  Found 7 models, 21 tests, 5 sources, 0 exposures, 0 metrics, 467 macros, 0 groups, 0 semantic models
14:05:37  
14:05:37  Concurrency: 1 threads (target='prod')
14:05:37  
14:05:37  1 of 7 START sql view model warehouse.stg_datas ................................ [RUN]
14:05:37  1 of 7 OK created sql view model warehouse.stg_datas ........................... [CREATE VIEW in 0.16s]
14:05:37  2 of 7 START sql view model warehouse.stg_diarias .............................. [RUN]
14:05:37  2 of 7 OK created sql view model warehouse.stg_diarias ......................... [CREATE VIEW in 0.04s]
14:05:37  3 of 7 START sql view model warehouse.stg_pagamentos ........................... [RUN]
14:05:37  3 of 7 OK created sql view model warehouse.stg_pagamentos ...................... [CREATE VIEW in 0.05s]
14:05:37  4 of 7 START sql table model warehouse.dim_tempo ............................... [RUN]
14:05:37  4 of 7 OK created sql table model warehouse.dim_tempo .......................... [SELECT 120 in 0.08s]
14:05:37  5 of 7 START sql table model warehouse.dim_elemento_despesa .................... [RUN]
14:05:37  5 of 7 OK created sql table model warehouse.dim_elemento_despesa ............... [SELECT 84 in 0.13s]
14:05:37  6 of 7 START sql table model warehouse.dim_favorecidos ......................... [RUN]
14:05:37  6 of 7 OK created sql table model warehouse.dim_favorecidos .................... [SELECT 433 in 0.06s]
14:05:37  7 of 7 START sql table model warehouse.fato_diarias_pagamentos ................. [RUN]
14:05:37  7 of 7 OK created sql table model warehouse.fato_diarias_pagamentos ............ [SELECT 602 in 0.07s]
14:05:37  
14:05:37  Finished running 3 view models, 4 table models in 0 hours 0 minutes and 0.70 seconds (0.70s).
14:05:37  
14:05:37  Completed successfully
14:05:37  
14:05:37  Done. PASS=7 WARN=0 ERROR=0 SKIP=0 TOTAL=7
[2023-09-17T14:05:39.109+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-09-17T14:05:39.118+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=etl, task_id=dbtRun, execution_date=20230917T140023, start_date=20230917T140531, end_date=20230917T140539
[2023-09-17T14:05:39.167+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-17T14:05:39.191+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
